app/assets/stylesheets/content/_forms.lsg

```diff
@@ -57,6 +57,35 @@
 </form>
 ```
 
+## Forms: Bordered style - Danger zone
+
+```
+<form class="form danger-zone">
+  <section class="form--section">
+    <h3 class="form--section-title">
+      Delete account <em>foo.bar@openproject.org</em>
+    </h3>
+    <p>
+      Your account will be deleted from the system. Therefore, you will no longer be able to log in with your current credentials. If you choose to become a user of this application again, you can do so by using the means this application grants.
+    </p>
+    <p>
+      Of the data you created as much as possible will be deleted. Note however, that data like work packages and wiki entries can not be deleted without impeding the work of the other users. Such data is hence reassigned to an account called "Deleted user".
+    </p>
+    <p class="danger-zone--warning">
+      <span class="icon icon-attention2"></span>
+      <span>Deleting your account is an irrevsible action</span>
+    </p>
+    <p>
+      Enter your login <em>foo.bar@openproject.org</em> to verfiy the deletion
+    </p>
+    <div class="danger-zone--verification">
+      <input type="text"></input>
+      <button class="button -highlight icon icon-delete" disabled>Delete</button>
```

Comment:
Correct me if I am wrong, but this is not how we do buttons with an icon, is it?

I remember something like extra wrappers for text and the icon... Looking at the screenshot the icon and the text are also not perfectly aligned on their base...